### ISSUE TITLE
Ran out of routes to try after 1 attempt

### ISSUE THREAD

**Issue by joelmackie** [shesek/spark-wallet#195]
I keep getting the error listed above every time I try to pay an invoice. 

Message in paystatus is: /home/username/.lightning/lingtning-rpc is missing, using the bitcoin mainnet subdirectory at /home/username/.lightning/bitcoin instead.  

Noob at Ubuntu but still learning. Any help would be greatly appreciated.

**Comment by shesek** [comment]
This is not directly related to Spark, the 'Ran out of routes to try' message is issued by c-lightning, which is unable to find a payment route to your payment destination.

> /home/username/.lightning/lingtning-rpc is missing, using the bitcoin mainnet subdirectory at /home/username/.lightning/bitcoin instead.

This is normal, you can safely ignore it.

But this shouldn't show up in relation to paystatus -- does it?

**Comment by joelmackie** [comment]
Yes it shows up when I enter spark-wallet --paystatus. Thank you for the feed back.
